views/application.py

```diff
@@ -1,17 +1,17 @@
 from rest_framework import response
-from rest_framework import viewsets, permissions
-
+from rest_framework import viewsets, permissions, generics
+from django.contrib.auth.hashers import check_password
```

Comment:
Give a line break after this. We follow a pattern for importing packages.
```
from external_library (django, restframework) import module

from external_app (open_auth, noticeboard, etc.) import module

from current_app (developer) import module
```

Notice the line breaks between each

Reply:
I think this check_password import is not being used anywhere, so remove the import statement